content/blog/2021-07-27-july-21-dvc-community-gems.md

```diff
@@ -0,0 +1,131 @@
+---
+title: July '21 Community Gems
+date: 2021-07-27
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: self-hosted runners, DVC commits,
+  troubleshooting remotes, and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: self-hosted runners, DVC commits,
+  troubleshooting remotes, and more.
+picture: 2021-07-27/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/july-21-community-gems/779
+tags:
+  - Community
+  - Runners
+  - Remotes
+  - CML
+  - DVC
+  - Gems
```

Comment:
Minor: so, do we use this or not :) I though that we decided that the one example we saw was made by mistake and we should fix that one.

Reply:
Yeah, we did decide that. 😅  I deleted the `Gems` tag from this and the other one to keep things consistent.